client/landing/stepper/declarative-flow/internals/steps-repository/importer-migrate-message/index.tsx

```diff
@@ -38,7 +39,24 @@ const ImporterMigrateMessage: Step = () => {
 	const fromUrl = useQuery().get( 'from' ) || '';
 	const siteSlug = siteSlugParam ?? '';
 	const isCredentialsSkipped = useQuery().get( 'credentials' ) === 'skipped';
-	const { isPending, sendTicket } = useSubmitMigrationTicket();
+	const [ hasError, setHasError ] = useState< APIError | null >( null );
+	const { isPending, sendTicket } = useSubmitMigrationTicket( {
```

Comment:
```suggestion
	const { isPending, sendTicket, hasError, error } = useSubmitMigrationTicket( {
```

You can capture the error state directly from the migration.